l10n/configs/pontoon.toml

```
@@ -213,6 +213,18 @@ locales = [
         "pt-PT",
         "nl",
     ]
+[[paths]]
+    reference = "en/firefox/whatsnew/whatsnew-115-vpn.ftl"
+    l10n = "{locale}/firefox/whatsnew/whatsnew-115-vpn.ftl"
+    locales = [
+        "cz",
```

```suggestion
        "cs",
```

This ain't a locale code 😉

---

Oops! Fixed.